### DISCUSSION TITLE
Crazy buttons

### DISCUSSION THREAD

**Discussion by ivnai** [fremtind/jokul#3704] [Annet]
Er i SB1 sin løsning på veien å kjøpe bil og båt forsikring, padding til både tekst og pil er ukonsekvent. Bare melder pirk.
<img width="2089" alt="Screenshot 2023-09-25 at 14 37 46" src="https://github.com/fremtind/jokul/assets/93194579/2a231c48-8452-4913-9daf-6587ad2dbfec">

**Comment by Stormoen** [comment]
@ivnai I kode eller figma?

**Reply by ivnai** [reply]
IRL, er påloget i Sparebank. Godt mulig at jeg meldte sak på feil adresse?

**Comment by ivarni** [comment]
Ser dette på https://kundeforsikring.sparebank1.no/privat/forsikring/motor/bil men ikke på https://jokul.fremtind.no/komponenter/buttons/

Vi hadde en endring på knapp-ikoner for ikke så lenge siden så det kan være at motor-teamet har noe CSS som går i beina på "vår" CSS kanskje?

**Comment by kennidenni** [comment]
@Stormoen @ivarni @ivnai 
Hehe er et kjent problem som vi har valgt å ikke gjøre noe med da bil nå blir flyttet til en ny applikasjon som ikke har noe ffe eller hacked-ffe stiler som lastes inn. Bil vil da bli lik f.eks båt. 

Men det løser ikke et problem som vi har i mobilbanken. Der har vi fortsatt problemer og vi tror det er SB1 som laster inn noe CSS i mobilbanken (uten at vi har sjekket det så veldig nøye). Du kan se at radio buttons ikke er helt på linje og at knappene ser litt rare ut. Det er problemer med alle PM produkter i mobilbanken og det har vært det i minimum 2 år.
![image](https://github.com/fremtind/jokul/assets/31621649/a58d7b29-0486-401a-af07-a03b3f22a232)

**Reply by ivnai** [reply]
OK! Ser funky ut :)

**Reply by kennidenni** [reply]
Ikke uenig 👀
